homeassistant/components/spider/climate.py

```diff
@@ -9,12 +9,23 @@
 
 from homeassistant.components.climate import (
     ATTR_TEMPERATURE, STATE_COOL, STATE_HEAT, STATE_IDLE,
-    SUPPORT_OPERATION_MODE, SUPPORT_TARGET_TEMPERATURE, ClimateDevice)
+    SUPPORT_OPERATION_MODE, SUPPORT_TARGET_TEMPERATURE,
+    SUPPORT_FAN_MODE, ClimateDevice)
 from homeassistant.components.spider import DOMAIN as SPIDER_DOMAIN
 from homeassistant.const import TEMP_CELSIUS
 
 DEPENDENCIES = ['spider']
 
+FAN_LIST = [
+    'Auto',
+    'Low',
+    'Medium',
+    'High',
+    'Boost 10',
```

Comment:
Is this really a fan mode?  Or does mean "increase the speed by 10"?  if it's the latter, this shouldn't go in the list of speeds, but perhaps would be a custom service that can be called to increase the speed.

Reply:
I must say that I do not know this setting since I don't have it personally. It's based on user feedback. However, all of these fan modes are configured the same way in spider. So either their interface and their API is weird or it's a real fan mode.

Reply:
Do you think you can work with someone who has this and who can test your changes?

Reply:
Yeah I did. See the linked issue in the description. It's fully working with someone who has tried the setup.

Let's put it different in regards to the fan modes; In the end I can only set it to one of these options. I cannot set it to `high` and `boost 10`. The same methods are being used in setting the fan mode and all modes were tested by a user. So splitting them is probably not recommended since it can only have one of these states.

Reply:
For you convenience: https://github.com/peternijssen/spiderpy/issues/3

Reply:
As long as `boost 10` is returned as the current mode, I'm all good.  What we want to avoid is a situation where these are options to set but not reflected in the running state.

Reply:
Yeah I totally understand! Whether or not the device does have this actual "action" or it's just a hack in their devices, the actual web interface and the way to set it, is through the fan mode.

Thanks for checking!